### DISCUSSION TITLE
Is there a way to know if a fed.brid.gy account is waiting for Atom/RSS or webmention?

### DISCUSSION THREAD

**Discussion by techmagus** [snarfed/bridgy-fed#945] [Q&A] [answered]
Hi,

Is there a way to know if a fed.brid.gy account is waiting for Atom/RSS polling schedule or webmention ping?

Or, maybe, a new feature where we can set if it should use Atom/RSS polling or webmention? (And while at it, to set which feed URL to use?)

Thank you!

(My site is `@youronly.one@youronly.one`.)

**Comment by snarfed** [comment, marked as answer]
Hi! Good idea. User pages like https://fed.brid.gy/web/youronly.one for web sites don't currently show whether we're using a feed or webmention, and they don't currently let you switch between them or specify a feed, but I can take those all as feature requests!

For your account specifically, it's currently still polling the feed from your home page. The last post it's seen is https://im.youronly.one/yuki/ko/kb/ddfon/federated-social-network-timeline-2022346/ .
